src/styles/blocks/price.scss

```diff
@@ -0,0 +1,53 @@
+.price {
+  &__wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    align-items: center;
+    margin-block: 24px 16px;
+    margin-inline: $margin-inline;
+  }
+
+  &__text {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $other-text-color;
+  }
+
+  &__money {
+    margin: 0;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__link {
+    display: block;
+    margin-inline: $margin-inline;
+
+    background-color: $button-color;
```

Comment:
The variable `$button-color` is used but not defined. Please define this variable to ensure the background color is applied correctly.